tests/pandas/test_continuous.py

```diff
@@ -0,0 +1,122 @@
+"""
+Contains unit tests for scores.continuous.standard
+"""
+# pylint: disable=missing-function-docstring
+# pylint: disable=line-too-long
+
+import numpy as np
+import pandas as pd
+import pytest
+
+import scores.pandas as scores
+
+PRECISION = 4
+
+# Mean Squared Error
+
+
+def test_mse_pandas_series():
+    """
+    Test calculation works correctly on pandas series
+    """
+
+    fcst_pd_series = pd.Series([1, 3, 1, 3, 2, 2, 2, 1, 1, 2, 3])
+    obs_pd_series = pd.Series([1, 1, 1, 2, 1, 2, 1, 1, 1, 3, 1])
+    expected = 1.0909
```

Comment:
Why not set up the tests to produce a nice number that you don't need to use `PRECISION = 4`? 

Having 11 integers in the pd series inputs doesn't really add much. You could achieve the same test with just 2 numbers. Perhaps add in floats, negative numbers, and a zero.

Reply:
Set up to match existing tests based on xarray,  I think the consistency is helpful. We could enhance the edge case testing in both places potentially, but at this point I think we're relying on the underlying implementation of mean() in pandas/xarray, which itself should be reliable.